beetsplug/bpd/__init__.py

```diff
@@ -334,7 +335,7 @@ def cmd_idle(self, conn, *subsystems):
 
     def cmd_kill(self, conn):
         """Exits the server process."""
-        exit(0)
+        sys.exit(0)
```

Comment:
Not sure why this is "safer"? I don't exactly object, but it doesn't seem too important either.

Reply:
There's a "Description" link next to the issue on Deepsource with some reasoning. I didn't research this any further, though.